WG.md

```diff
@@ -8,12 +8,43 @@ for review by the TSC at an agreed upon date. A WG will be automatically disband
 
 ## Current Working Groups
 
+### Generics and circuit families
+
+**Objective**: ...
```

Comment:
While I haven't had the opportunity to attend a TSC meeting, I don't believe we have been handed down a specific remit from above. My understanding is that our purpose is to recommend what range of generics / parameterised circuit family functionality (beyond angle parameters) would be suitable for inclusion in OpenQASM, and what syntax that should be used for it.